src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -16,8 +16,6 @@ namespace System.Diagnostics
 {
     public partial class Process : IDisposable
     {
-        private static readonly UTF8Encoding s_utf8NoBom =
-            new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
```

Comment:
The problem with this change is that you're now specifying the default IO encoding in three separate locations. You might want to do something like
```csharp
private static readonly UTF8Encoding s_defaultEncoding = Encoding.Default;
```